pkg/xds/bootstrap/generator.go

```diff
@@ -127,6 +130,10 @@ func (b *bootstrapGenerator) Generate(ctx context.Context, request types.Bootstr
 		UseDeltaXds:          b.deltaXdsEnabled,
 	}
 
+	if b.dynamicLoopbackOutbounds {
+		params.Features[xds_types.FeatureDynamicLoopbackOutbounds] = true
+	}
+
 	if params.Features.HasFeature(xds_types.FeatureDeltaGRPC) {
```

Comment:
I think we should go with a DP feature, it's easy to add this to your ECS task and it's a more flexible setup.

The feature could be named: "no-passthrough"

Reply:
I've implemented here 2 ways:
* Single DP can enable feature
* We can enable feature for all DPs in this case user don't need to define it on each DP